packages/inventory/src/Stocktake/api/api.ts

```diff
@@ -3,48 +3,28 @@ import {
   formatNaiveDate,
   InsertStocktakeLineInput,
   UpdateStocktakeLineInput,
-  ConnectorError,
-  OmSupplyApi,
-  StocktakeQuery,
-  StocktakeLineNode,
-  StocktakeLineConnector,
   UpdateStocktakeInput,
   RecordPatch,
+  SortBy,
```

Comment:
pretty much exactly like requisitions. Api file and hooks file updates..

Reply:
and removal of the Guards 😁